makefiles/pseudomodules.inc.mk

```
@@ -247,6 +247,7 @@ PSEUDOMODULES += sx1261
 PSEUDOMODULES += sx1262
 PSEUDOMODULES += sx1268
 PSEUDOMODULES += llcc68
+PSEUDOMODULES += sx126x_%
```

```suggestion
PSEUDOMODULES += sx126x_stm32wl
PSEUDOMODULES += sx126x_rf_switch
```

And please add some lines of documentation for `sx127x_rf_switch`.